src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoLongDatePattern.cs

```diff
@@ -30,6 +30,17 @@ public void PosTest2()
             VerificationHelper(new DateTimeFormatInfo(), "dddd, dd MMMM yyyy", true);
         }
 
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("es-ES")]
+        public void TestCultureLongDate(string cultureName)
+        {
+            CultureInfo ci = new CultureInfo(cultureName);
+            string expected = DateTimeFormatInfoData.GetLongDatePattern(ci);
```

Comment:
Should this variable be named 'actual' instead of 'expected'? I've changed a few on my side as well